Userland/Libraries/LibWeb/SVG/SVGLinearGradientElement.h

```diff
@@ -55,8 +55,6 @@ class SVGLinearGradientElement : public SVGGradientElement {
     Optional<NumberPercentage> m_y1;
     Optional<NumberPercentage> m_x2;
     Optional<NumberPercentage> m_y2;
-
-    mutable RefPtr<Gfx::SVGLinearGradientPaintStyle> m_paint_style;
```

Comment:
...which ties into the fact that you're no longer caching this?

Reply:
I reverted this cache along with making `Painting::PaintStyle` ref-counted.